cuqi/experimental/mcmc/_langevin_algorithm.py

```diff
@@ -22,6 +80,14 @@ def validate_target(self):
         except (NotImplementedError, AttributeError):
             raise ValueError("The target need to have a gradient method")
 
+    def _accept_or_reject(self, x_star, target_eval_star, target_grad_star):
+        # accept directly
```

Comment:
Could you create a docstring for this method that explains what it does? e.g. that it stores provided proposals in the sampler state (current_point, current_target_eval etc.?)

Reply:
Done